homeassistant/components/__init__.py

```diff
@@ -22,6 +23,15 @@ def is_on(hass: HomeAssistant, entity_id: str | None = None) -> bool:
 
     If there is no entity id given we will check all.
     """
+    report(
+        (
+            "uses homeassistant.components.is_on."
+            " This is deprecated and will stop working in Home Assistant 2024.6, it"
+            " should be updated to use the function of the platform directly."
```

Comment:
Is there a specific reason or rush to divert from the default of 6 months?

Reply:
No, not really. I didn't find any usage, so I thought that a period of 3 months would be enough, but of course we can go with the default of 6 months.

Reply:
If there is no reason not to use the default, we should just use the default IMHO.

Reply:
Okay, will change this 👍